### DISCUSSION TITLE
沒辦法設定IP length

### DISCUSSION THREAD

**Discussion by tofu31** [bol-edu/HLS-SOC-Discussions#211] [Q&A]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/46846610/21dc0288-e310-414e-a8b1-6ff49c846593)

4和8可以寫，但改成12就會fail，該怎麼處理

**Comment by michael61112** [comment]
Hi 同學，可以多描述一些細節嗎? 沒有看出你的標題和截圖、內文有什麼關係??
